web-animations-2/Overview.bs

```diff
@@ -805,6 +850,55 @@ The procedure to [=pause an animation=] needs to refer not only to the
 <ins cite="#amend-CE">Likewise, the procedure to [=pause an animation=] needs to include scheduling
 a task for updating [=custom effects=].</ins>
 
+Update the constraints for setting the seek time to only apply when using a
+monotonic timeline. A [=scroll-driven animation=] needs to defer setting the
+hold time until the animation range has been computed.
+
+Replace:
+
+> If |seek time| is [=unresolved|resolved=],
+>
+>   <dl class=switch>
+>
+>       : If |has finite timeline| is true,
+>       ::
+>           Set |animation|'s [=animation/start time=] to |seek time|.
+>
+>       : Otherwise,
+>       ::
+>           Set |animation|'s [=hold time=] to |seek time|.
+>
+>   </dl>
+
+With:
+
+> If |seek time| is [=unresolved|resolved=], and
+> |has finite timeline| is false,
+>    *  Set |animation|'s [=hold time=] to |seek time|.
```

Comment:
Actually, I'm still confused about this. This will set the hold time to 0 or 100% if we pause with an unresolved current time. When would we want this behavior?

Reply:
A time based animation starting from an idle state. The [==unresolved\resolved=] label is likely the source of confusion.  The state is actually resolved, but links to text describing when the state would be unresolved.

Reply:
Removed seek time from pausing-an-animation for better alignment with playing-an-animation.

Reply:
Thanks this reads much more clearly now.